internal/beater/beater.go

```diff
@@ -685,8 +685,7 @@ func (s *Runner) newFinalBatchProcessor(
 	memLimit float64,
 ) (modelpb.BatchProcessor, func(context.Context) error, error) {
 	if s.elasticsearchOutputConfig == nil {
-		monitoring.Default.Remove("libbeat")
-		libbeatMonitoringRegistry := monitoring.Default.NewRegistry("libbeat")
+		libbeatMonitoringRegistry := monitoring.Default.GetRegistry("libbeat")
```

Comment:
Q: Is this related to the fix? I was under the impression that we specifically remove and create a new libbeat repo to workaround EA hot reload where 2 final processors run in parallel